packages/config/README.md

```diff
@@ -284,6 +302,19 @@ _Type_: `object?`
 
 When the [`buffer`](#buffer) option is used, this contains two arrays `stdout` and `stderr` with the logs.
 
+#### env
+
+_Type_: `object?`
+
+Site's environment variables. Each property is an object of key-value pairs:
+
+- `general`: general environment variables set for all sites
+- `account`: environment variables set in the Netlify UI for a specific account
```

Comment:
The term `account` is widely used in `netlify/build`, but CLI uses `team`. Should we try to standardise this? If so, which one should we adopt?

Reply:
Yes we should, great point :) We refer to those publicly as team:
![image](https://user-images.githubusercontent.com/26760571/104013777-a04b7a00-51ba-11eb-947d-0cbf8bc6e103.png)

So it make sense to me to use that term. @ehmicky WDYT?

Reply:
Good point! I opened an issue at https://github.com/netlify/build/issues/2164